src/helpers/map.js

```diff
@@ -124,40 +198,35 @@ export function traversableCellsInTile(tileIsTraversable, cpt, objRadius) {
 
 /*
  * Returns a 2D array of traversable (1) and blocked (0) cells. Size of return grid is
- * map.length * cpt
+ * map.length * CPTL
  *
  * The 2D array is an array of the columns in the game. empty_tiles[0] is
  * the left-most column. Each column array is an array of the tiles in
  * that column, with 1s and 0s.  empty_tiles[0][0] is the upper-left corner
  * tile.
  *
- * @param {number} cpt - number of cells per tile
  * @param {number} map - 2D array representing the Tagpro map
  */
-export function getTraversableCells(cpt, map) {
+export function getMapTraversabilityInCells(map) {
```

Comment:
Hmmm... yeah, `getMapTraversabilityInCells` probably isn't perfect. Though, the reason I changed it from `getTraversable...` is because you aren't just getting the traversable cells, you are getting all the cells and seeing if they are traversable. If I had a function called `getMapTraversableCells` I would have it return the positions of all the traversable cells or something like that. I just want the name to be as self-documenting as possible. Thoughts?